.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-## Background
-<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+## Description of change
+Please include a description of the change. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary.
```

Comment:
Do we want to call out links to relevant issues?

Reply:
It's hard to come up with a universal set of requirements for this, and I fully anticipate making changes as we go along; some things to stew on:

1) does this change modify integrations with third party systems?
2) is there a feature flag for this change, and if so, does the devops team need to e made aware of varying configuration for each environment?
3) are there any backwards-incompatible changes for consumers of the API, and have all API modifications been reflected in the swagger documentation?
4) are all requests asynchronous or the reasons for them not being so documented?
5) are there dashboards and associated instrumentation in place for this change? Have alerts been configured? 
6) Which alerts would indicate an issue with this functionality?